openreview/conference/builder.py

```diff
@@ -397,9 +397,9 @@ def get_homepage_options(self):
             options['deadline'] = self.homepage_header.get('deadline')
         return options
 
-    def get_submissions(self, accepted = False, details = None):
+    def get_submissions(self, accepted = False, details = None, sort = 'number:asc'):
```

Comment:
I prefer to have `sort=None` as a default parameter and specify the sort only when we need it.